app/lib/lib/roomTypes/private.js

```diff
@@ -71,9 +71,6 @@ export class PrivateRoomType extends RoomTypeConfig {
 
 	canSendMessage(roomId) {
 		const room = ChatRoom.findOne({ _id: roomId, t: 'p' }, { fields: { prid: 1 } });
```

Comment:
Hey, `room` is never used, you should delete it. That's the reason your CI is failing.